packages/app/src/runner/SpecRunnerHeader.vue

```diff
@@ -1,54 +1,117 @@
+          :disabled="isDisabled"
+          class="rounded-none border-r-1px border-gray-100 mr-12px"
+          variant="text"
+          @click="togglePlayground"
         >
-          <div>
-            {{ autStore.url }}
-          </div>
+          <i-cy-crosshairs_x16 class="icon-dark-gray-600" />
+        </Button>
+        <div class="mr-12px max-w-100% grid text-gray-600 items-center truncate">
+          {{ autStore.url }}
         </div>
-
-        <div>Loading URL: {{ autStore.isLoadingUrl }}</div>
       </div>
-
-      <Select
-        :model-value="browser"
+      <div
+        v-else
+        class="flex-grow"
+      >
+        <Button
+          data-cy="playground-activator"
+          :disabled="isDisabled"
+          class=" border-gray-100 mr-12px"
+          variant="outline"
+          @click="togglePlayground"
+        >
+          <i-cy-crosshairs_x16 class="icon-dark-gray-600" />
+        </Button>
+      </div>
+      <SpecRunnerDropdown
+        v-if="selectedBrowser?.displayName"
         data-cy="select-browser"
-        :options="browsers"
-        item-value="displayName"
-        @update:model-value="changeBrowser"
-      />
+      >
+        <template #heading>
+          <img
+            class="min-w-16px w-16px"
+            :src="allBrowsersIcons[selectedBrowser.displayName]"
+          > {{ selectedBrowser.displayName }} {{ selectedBrowser.majorVersion }}
+        </template>
+
+        <template #default>
+          <div class="max-h-50vh overflow-auto">
+            <VerticalBrowserListItems
+              :gql="props.gql"
+              :spec-path="activeSpecPath"
+            />
+          </div>
+        </template>
+      </SpecRunnerDropdown>
+      <SpecRunnerDropdown
+        variant="panel"
+        data-cy="viewport"
+      >
+        <template #heading>
+          <i-cy-ruler_x16 class="icon-dark-gray-500 icon-light-gray-400" />
+          <span class="whitespace-nowrap">{{ autStore.viewportWidth }}x{{ autStore.viewportHeight }}</span>
+          <span
+            v-if="displayScale"
+            class="-mr-6px text-gray-500"
+          > ({{ displayScale }})</span>
+        </template>
+        <template #default>
+          <div class="max-h-50vw p-16px text-gray-600 leading-24px w-400px overflow-auto">
+            <!-- TODO: This copy is a placeholder based on the old message for this, we should confirm the exact copy and then move to i18n -->
+            <p class="mb-16px">
+              The
+              <strong>viewport</strong> determines the width and height of your application.
+              By default the viewport will be
+              <strong>{{ autStore.defaultViewportWidth }}px</strong> by
+              <strong>{{ autStore.defaultViewportHeight }}px</strong> for {{ props.gql.currentTestingType === "e2e" ? 'End-to-end' : 'Component' }}
+              Testing unless specified by a <InlineCodeFragment>cy.viewport</InlineCodeFragment> command.
```

Comment:
Does this mean the viewport can no longer be set in the cypress configuration?

Reply:
This is pretty much the same text we have currently, and I do think it could be improved. There's no content for this in Figma, so I've brought the content over from 9.x as a starting point. There are multiple ways to set viewport, and it can be different for each testing type, so my guess is, this will end up much shorter and link out to the docs. But not worth holding up the PR to get this content.